docker-hub/vulnerability-scanning.md

```diff
@@ -10,15 +10,34 @@ title: Hub Vulnerability Scanning
   target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_scan"
 %}
 
-Docker Hub Vulnerability Scanning enables you to automatically scan Docker images for vulnerabilities using Snyk. This uses the same technology as the [docker scan](../engine/scan/index.md) command.
+Docker Hub Vulnerability Scanning enables you to automatically scan Docker
+images for vulnerabilities using Snyk. This uses the same technology as the
+[docker scan](../engine/scan/index.md) command.
+
+> When you enable Hub Vulnerability Scanning, you can also see whether your
+> images are affected by Log4Shell (CVE-2021-44228). For more information, see
+> [Scan images](#scan-images).
 
 When you push an image to Docker Hub after enabling vulnerability scanning, Docker Hub automatically scans the image to identify vulnerabilities in your container images. Vulnerability Scanning allows developers and development teams to review the security state of the container images and take actions to fix issues identified during the scan, resulting in more secure deployments. The scan result includes the source of the vulnerability, such as OS packages and libraries, version in which it was introduced, and a recommended fixed version (if available) to remediate the vulnerabilities discovered.
 
 ## Scan images
 
```

Comment:
this should be "Scan images on Docker Hub" or something

Reply:
Updated